Compile error on Windows

### Component(s)

_No response_

### What happened?

```
ok  	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/networkscraper	1.288s
# github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/pagingscraper [github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/pagingscraper.test]
Error: internal\scraper\pagingscraper\paging_scraper_windows_test.go:102:18: metricsConfig.SystemPagingUtilization undefined (type metadata.MetricsBuilderConfig has no field or method SystemPagingUtilization)
Error: internal\scraper\pagingscraper\paging_scraper_windows_test.go:174:18: metricsConfig.SystemPagingUtilization undefined (type metadata.MetricsBuilderConfig has no field or method SystemPagingUtilization)
```

https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/4348660778/jobs/7597443049



### Collector version

latest

### Environment information

## Environment
OS: (e.g., "Ubuntu 20.04")
Compiler(if manually compiled): (e.g., "go 14.2")


### OpenTelemetry Collector configuration

_No response_

### Log output

_No response_

### Additional context

_No response_

--- a/receiver/hostmetricsreceiver/internal/scraper/pagingscraper/paging_scraper_windows_test.go
+++ b/receiver/hostmetricsreceiver/internal/scraper/pagingscraper/paging_scraper_windows_test.go
@@ -99,7 +99,7 @@ func TestScrape_Errors(t *testing.T) {
 	for _, test := range testCases {
 		t.Run(test.name, func(t *testing.T) {
 			metricsConfig := metadata.DefaultMetricsBuilderConfig()
-			metricsConfig.SystemPagingUtilization.Enabled = true
+			metricsConfig.Metrics.SystemPagingUtilization.Enabled = true
 
 			scraper := newPagingScraper(context.Background(), receivertest.NewNopCreateSettings(), &Config{MetricsBuilderConfig: metricsConfig})
 			if test.getPageFileStats != nil {
@@ -171,7 +171,7 @@ func TestStart_Error(t *testing.T) {
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
 			metricsConfig := metadata.DefaultMetricsBuilderConfig()
-			metricsConfig.SystemPagingUtilization.Enabled = true
+			metricsConfig.Metrics.SystemPagingUtilization.Enabled = true
 
 			scraper := newPagingScraper(context.Background(), receivertest.NewNopCreateSettings(), &Config{MetricsBuilderConfig: metricsConfig})
 

